### ISSUE TITLE
Add black stroke around skater's red position dot

### ISSUE THREAD

**Issue by arouinfar** [phetsims/energy-skate-park#467]
For #446 

The skater's position dot does not always have sufficient contrast against its background.

<img width="118" height="47" alt="Image" src="https://github.com/user-attachments/assets/abbde326-8fc8-4f02-83d7-01b8c7e3da63" />

<img width="128" height="40" alt="Image" src="https://github.com/user-attachments/assets/e0b5da39-7537-4b23-990c-e23ed52b0e2a" />

<img width="120" height="61" alt="Image" src="https://github.com/user-attachments/assets/d337abd8-4b1a-4b0b-8719-1c797a260764" />

@samreid can you please add a black stroke around it? The weight can match the stroke used for the path dots.

**Comment by samreid** [comment]
<img width="614" height="263" alt="Image" src="https://github.com/user-attachments/assets/611e69d6-35b8-4340-90cf-4843444fca4d" />
